.circleci/config.yml

```diff
@@ -1,17 +1,17 @@
-# https://circleci.com/docs/2.0/configuration-reference/
 version: 2.1
+orbs:
+  node: circleci/node@1.1.6
 jobs:
-  build:
-    docker:
-      - image: circleci/node:12
+  build-and-test:
+    executor:
+      name: node/default
     steps:
       - checkout
-      - run:
-          name: "Install node packages"
-          command: npm install
-      - run:
-          name: "Run linting"
-          command: npm run lint
-      - run:
-          name: "Run testing"
-          command: npm run test
+      - node/with-cache:
```

Comment:
using orbs - this is cool and advanced ... i give you small challenge now, try to make this CI so that workflow contains from two jobs - one is running lint and 2'nd test